src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -59,6 +59,17 @@ def collect_tools_from_directory(base_dir) -> dict:
     return tools
 
 
+def set_default_input_index_ui_hint(tool):
+    inputs_dict = tool["inputs"]
+    input_index = 0
+    for input_name, settings in inputs_dict.items():
+        if "input_type" in settings.keys() and settings["input_type"] == "uionly_hidden":
```

Comment:
Add comment here to explain why we need to skip this case

Reply:
Added, thanks.